Build is broken in CI

During `mono bootstrap --ci`, calling `npm ci` seems to result in failure:

```
Bootstrapping project
npm ci
npm ERR! code ERR_FS_EISDIR
npm ERR! syscall rm
npm ERR! path /home/semaphore/appsignal-nodejs/node_modules/.bin
npm ERR! errno 21
npm ERR! Path is a directory: rm returned EISDIR (is a directory) /home/semaphore/appsignal-nodejs/node_modules/.bin
```

Here's a failing build: https://appsignal.semaphoreci.com/jobs/5774ab88-1843-4244-b728-ead9383b5e15

--- a/.semaphore/semaphore.yml
+++ b/.semaphore/semaphore.yml
@@ -27,7 +27,7 @@ global_job_config:
     - checkout
     - '[ -n "$NODE_VERSION" ] && sem-version node $NODE_VERSION || echo Skipping Node.js
       install'
-    - npm i -g npm@latest
+    - npm i -g npm@9.2.0
     - script/setup
     - source ~/.bashrc
 blocks:

--- a/build_matrix.yml
+++ b/build_matrix.yml
@@ -22,7 +22,7 @@ semaphore: # Default `.semaphore/semaphore.yml` contents
       commands:
         - checkout
         - "[ -n \"$NODE_VERSION\" ] && sem-version node $NODE_VERSION || echo Skipping Node.js install"
-        -  npm i -g npm@latest
+        -  npm i -g npm@9.2.0
 
         # Mono setup
         - script/setup

